src/fillTank.test.js

```diff
@@ -1,11 +1,58 @@
-'use strict';
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
+  const customer = {
+    money: 3000,
+    vehicle: {
+      maxTankCapacity: 40,
+      fuelRemains: 8,
+    },
+  };
+  const fuelPrice = 4.8;
 
-  it('should ', () => {
+  it("should be declared", () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it("should round the poured amount to the tenth part", () => {
+    fillTank(customer, fuelPrice, 20.78);
```

Comment:
The test 'should round the poured amount to the tenth part' has a mutation. It changes the initial state of the customer object. It would be better to create a separate customer object for this particular test.